plugins/tech-insights-backend/src/service/fact/FactRetrieverEngine.ts

```diff
@@ -48,7 +48,7 @@ export interface FactRetrieverEngine {
    * Schedules fact retriever run cycles based on configuration provided in the registration.
    *
    * Default implementation uses backend-tasks to handle scheduling. This function can be called multiple
-   * times, where initial calls schedule the tasks and subsequents invocations update the schedules.
+   * times, where initial calls schedule the tasks and subsequent invocations update the schedules.
```

Comment:
This doc comment actually does ship in the package. But no worries, it can go out as part of something else down the line